.github/scripts/check_labels.py

```diff
@@ -27,8 +27,8 @@
 )
 
 
-def get_release_notes_labels() -> List[str]:
-    return [label for label in get_pytorch_labels() if label.lstrip().startswith("release notes:")]
+def get_release_notes_labels(org: str, proj: str) -> List[str]:
+    return [label for label in gh_get_labels(org, proj) if label.lstrip().startswith("release notes:")]
```

Comment:
s/proj/repo/

```suggestion
def get_release_notes_labels(org: str, repo: str) -> List[str]:
    return [label for label in gh_get_labels(org, repo) if label.lstrip().startswith("release notes:")]
```

Reply:
Happy to change to `repo`, but just want to highlight the reason `proj` was chosen was to stay consistent with GitHubPR class attribute: https://github.com/pytorch/pytorch/blob/master/.github/scripts/trymerge.py#L674

Reply:
No need for a todo, but generally github's official docs call this bit the repository name, with the "owner/repo_name" collectively called the "repository"

At least when opportunities come up, it would be good to shift to a naming convention that matches that.